app/Models/Campaign.php

```diff
@@ -298,6 +298,11 @@ public function toSearchableArray()
             ]);
         });
 
+        // Transform dates into UNIX timestamps:
+        foreach ($this->dates as $date) {
```

Comment:
Is `$this->dates` a `Searchable` method that automatically includes all date fields?

Reply:
That's a [property on the `Campaign` model](https://github.com/DoSomething/rogue/blob/306e88221fa34510133786f1f4b3dc314dedaa10/app/Models/Campaign.php#L18-L23) (and any Eloquent model)! It's used by Laravel to know which fields to parse from the database into `Carbon` instances.